src/Service/Context/PageContext.php

```diff
@@ -94,7 +95,9 @@ public function __construct(ConfigFactoryInterface $config_factory, EntityTypeMa
    */
   private function setPageContextCredential($credential_settings) {
     foreach (SELF::$CREDENTIAL_MAPPING as $credential_key => $tag_name) {
-      $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      if (isset($credential_settings[$credential_key]) {
+        $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      }
```

Comment:
I think we should always set the pageContext, but default to empty string. This way, a meta tag (although can be empty value) is always dropped on the page.